com.unity.render-pipelines.universal/Documentation~/universalrp-asset.md

```diff
@@ -40,11 +40,12 @@ These settings control the quality level of the URP. This is where you can make
-| __MSAA__         | Use [Multi Sample Anti-aliasing](https://en.wikipedia.org/wiki/Multisample_anti-aliasing) by default for every Camera in your Scene while rendering. This softens edges of your geometry, so they’re not jagged or flickering. In the drop-down menu, select how many samples to use per pixel: __2x__, __4x__, or __8x__. The more samples you choose, the smoother your object edges are. If you want to skip MSAA calculations, or you don’t need them in a 2D game, select __Disabled__. You can override this for individual cameras in the Camera Inspector. |
-| __Render Scale__ | This slider scales the render target resolution (not the resolution of your current device). Use this when you want to render at a smaller resolution for performance reasons or to upscale rendering to improve quality.  This only scales the game rendering. UI rendering is left at the native resolution for the device. |
+| Property                  | Description                                                  |
+| ------------------------- | ------------------------------------------------------------ |
+| __HDR__                   | Enable this to allow rendering in High Dynamic Range (HDR) by default for every camera in your Scene. With HDR, the brightest part of the image can be greater than 1. This gives you a wider range of light intensities, so your lighting looks more realistic. With it, you can still see details and experience less saturation even with bright light. This is useful if you want a wide range of lighting or to use [bloom](https://docs.unity3d.com/Manual/PostProcessing-Bloom.html) effects. If you’re targeting lower-end hardware, you can disable this to skip HDR calculations and get better performance. You can override this for individual cameras in the Camera Inspector. |
+| __MSAA__                  | Use [Multi Sample Anti-aliasing](https://en.wikipedia.org/wiki/Multisample_anti-aliasing) by default for every Camera in your Scene while rendering. This softens edges of your geometry, so they’re not jagged or flickering. In the drop-down menu, select how many samples to use per pixel: __2x__, __4x__, or __8x__. The more samples you choose, the smoother your object edges are. If you want to skip MSAA calculations, or you don’t need them in a 2D game, select __Disabled__. You can override this for individual cameras in the Camera Inspector. |
+| __Render Scale__          | This slider scales the render target resolution (not the resolution of your current device). Use this when you want to render at a smaller resolution for performance reasons or to upscale rendering to improve quality.  This only scales the game rendering. UI rendering is left at the native resolution for the device. |
+| __Scale Post-processing__ | Enable this to allow some post-processing effects to be done downscaled at the render scale resolution. This can improve frame times at the cost of increased bandwidth. |
```

Comment:
"some post-processing effects" - can we list those effects? Or is there a property (of an effect) that determines whether this scaling is applicable?
My suggestion:
"When this property is enabled, Unity downscales certain post-processing effects at the render scale resolution. This might improve frame times at the cost of increased VRAM bandwidth. The effects to which this property applies are: ..."

Reply:
It applies to the post-processing effects being applied in the final post-processing pass. If FXAA is enabled, the final pass is FXAA+dither+film grain. If FXAA is disabled, it's the uber-pass (bloom, chromatic abberation, film grain, dither, color grading/tonemapping, vignette).

Reply:
I would suggest the following text then:
"When this property is enabled, Unity downscales certain post-processing effects using the render scale resolution. This property affects the following effects: bloom, chromatic aberration, tone mapping, film grain, FXAA, dithering, and vignette.<br/>Enabling this option might improve frame times at the cost of increased VRAM bandwidth."

Reply:
The code snipped above wasn't updated when I pushed a change. The current text is:
"Enable this to allow some post-processing effects to be done downscaled at the render scale resolution. This can improve frame times at the cost of increased GPU bandwidth. The following effects are affected: bloom, chromatic aberration, tonemapping, film grain, FXAA, dithering and vignette."

Reply:
Thanks for the update (and sorry for the delay in replying), I would still suggest this text, if it's technically correct:
"When this property is enabled, Unity downscales certain post-processing effects using the render scale resolution. This property affects the following effects: bloom, chromatic aberration, tone mapping, film grain, FXAA, dithering, and vignette.
Enabling this option might improve frame times at the cost of increased VRAM bandwidth."